docs/source/conf.py

```diff
@@ -23,7 +23,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.pngmath', 'sphinxcontrib.tikz']
+extensions = ['sphinx.ext.pngmath', 'sphinxcontrib.tikz', 'sphinx.ext.numfig']
```

Comment:
Which sphinx-numfig were you using? It looks like the [latest one](https://pypi.python.org/pypi/sphinx_numfig) goes under the name sphinx_numfig (hence that is what would need to be listed here)